src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -34,10 +34,12 @@ public class BicepCompletionProvider : ICompletionProvider
         private static readonly Container<string> PropertyAccessCommitChars = new Container<string>(".");
 
         private IFileResolver FileResolver;
+        private IResourceSnippetsProvider ResourceSnippetsProvider;
```

Comment:
>private [](start = 8, length = 7)

nit - `readonly`

Reply:
Updated as part of below commit. Thanks!

https://github.com/Azure/bicep/commit/91ec6ff34132d0b6be98410a28548453a5b15375